src/dc_job.rs

```diff
@@ -712,7 +709,7 @@ unsafe fn dc_send_mdn(context: &Context, msg_id: uint32_t) {
         timestamp: 0,
         rfc724_mid: 0 as *mut libc::c_char,
         loaded: DC_MF_NOTHING_LOADED,
-        msg: 0 as *mut dc_msg_t,
+        msg: dc_msg_new(context, Viewtype::Unknown),
```

Comment:
Now msg is initialized twice, `dc_mimefactory_init` does the same again. Not a request to change, just a note. It will be fixed once `dc_mimefactory_t` is refactored to have `::new`.